core/src/main/scala/org/apache/spark/api/java/JavaSparkContext.scala

```diff
@@ -270,7 +272,9 @@ class JavaSparkContext(val sc: SparkContext)
    * }}}
    *
    * Do
-   * `JavaPairRDD<String, byte[]> rdd = sparkContext.dataStreamFiles("hdfs://a-hdfs-path")`,
+   * {{{
+   *   JavaPairRDD<String, byte[]> rdd = sparkContext.dataStreamFiles("hdfs://a-hdfs-path")
+   * }}},
```

Comment:
note to myself - see https://github.com/apache/spark/pull/15999#discussion_r89452453. It seems we always should use `{{{...}}}` instead of backticks or `<code>...</code>` to print out `<` and `>` for both scaladoc and javadoc.

It is fine now.

<img width="856" alt="2016-11-25 5 40 31" src="https://cloud.githubusercontent.com/assets/6477701/20619277/fa41be10-b336-11e6-8948-e5e07da3e99b.png">
<img width="705" alt="2016-11-25 5 41 09" src="https://cloud.githubusercontent.com/assets/6477701/20619278/fa466e56-b336-11e6-89fa-641bbec23753.png">